GenderPayGap.WebUI/Controllers/HomeController.cs

```diff
@@ -85,6 +85,16 @@ public async Task<IActionResult> PrivacyPolicyPost()
         }
 
         #endregion
+        
+        #region Accessibility Statement
+        
+        [HttpGet("~/accessibility-statement")]

```

Comment:
I think we don't need the `~`

It's only useful if we plan to host the site on a sub-folder (e.g. if we changed from gender-pay-gap.service.gov.uk to gov.uk/gender-pay-gap-service)

We don't plan to do this and it would break loads of other things too, so it's probably best to simplify this to `/accessibility-statement`